_posts/2018-01-19-nzoi-january-camp-2018.markdown

```diff
@@ -23,4 +23,4 @@ Thanks again to all the lecturers and tutors involved, thanks to Tom for help wi
 
 * * *
 
-If you're a student who didn't go to camp, but who thinks this sounds fun, then participate in [NZIC](nzoi.org.nz/nzic)! It's a free online competition, run throughout the year, with some problems approachable to newcomers and some hard ones to work towards. NZIC participants will be contacted towards the end of the year about selection for January Camp 2019.
+If you're a student who didn't go to camp, but who thinks this sounds fun, then participate in [NZIC](../nzic)! It's a free online competition, run throughout the year, with some problems approachable to newcomers and some hard ones to work towards. NZIC participants will be contacted towards the end of the year about selection for January Camp 2019.
```

Comment:
I think this should be a root-relative link (`/nzic`), otherwise https://www.nzoi.org.nz/2018/01/19/nzoi-january-camp-2018/ will link to https://www.nzoi.org.nz/2018/01/19/nzic